west.yml

```diff
@@ -92,6 +92,7 @@ manifest:
     - name: mbedtls
       revision: 9638398d3c319074ae3878035138ef8a76001e5b
       path: modules/crypto/mbedtls
+      revision: 8046c56542f8db737b1d35b9c21b3bd9a5db6d1b
```

Comment:
This looks wrong, you should be replacing line 93 not adding a new line.

Reply:
Ouch, indeed. I messed up the merge conflict, dunno what I was thinking. Anyway, it is fixed now.